src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/Details/components/DetailInput.tsx

```diff
@@ -9,6 +9,7 @@ interface DetailInputProps {
   field?: RegularFieldType
   formikProps: FormikProps<EditQuoteInput>
   nameRoot:
+    | ''
```

Comment:
I went back and forth between having this be optional, and just passing name instead. But I thought that It might be more consistent with how it's used. And forcing you to pass a root field.